container_api/src/error.rs

```diff
@@ -29,6 +31,7 @@ impl Error for HolochainInstanceError {
             HolochainInstanceError::InternalFailure(ref err)  => Some(err),
             HolochainInstanceError::InstanceNotActiveYet => None,
             HolochainInstanceError::InstanceAlreadyActive => None,
+            HolochainInstanceError::NoSuchInstance => None,
```

Comment:
How hard would it be to add an ID to these three errors? (InstanceNotActiveYet, InstanceAlreadyActive, NoSuchInstance)? These parameterless errors are tough to track. I could believe that it's hard to add, but just asking.